packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -97,8 +112,12 @@ const MT5TradeScreen: FC<MT5TradeScreenProps> = ({ mt5Account }) => {
                         </div>
                     </div>
                     <div className='wallets-mt5-trade-screen__details-description--right'>
-                        <WalletText weight='bold'>{details?.display_balance}</WalletText>
-                        {showNoNewPositionsMessage}
+                        {!(
```

Comment:
optional - probably worth refactoring this conditions to something easier to read, or at least adding a tests, as this condition is really complex. Good indicatore for complexity is if it can be explained to someone. For the below, it could be:
"check if platform is mt5 and some mapper contains an object of account with market type matching current market type and the status of given object is 'migrated_without_position', and if any of those conditions are NOT true then display wallet balance"